common/src/main/resources/assets/voicechat/lang/ko_kr.json

```diff
@@ -1,217 +1,212 @@
-  "message.voicechat.onboarding.voice.description": "마이크 테스트를 활성화하고 볼륨이 조용할 때는 슬라이더 아래에 있고 말을 할 때는 슬라이더 위에 있는지 확인하세요.",
-  "message.voicechat.onboarding.voice.title": "마이크 활성화 임계값 조정",
-  "message.voicechat.open": "열기",
-  "message.voicechat.open_folder": "폴더 열기",
-  "message.voicechat.optional_password": "암호 (선택)",
-  "message.voicechat.other_volume": "기타",
-  "message.voicechat.other_volume.description": "플레이어가 아닌 대상 소리 볼륨",
-  "message.voicechat.password": "암호",
-  "message.voicechat.ping_received": "%sms 안에 응답을 수신함",
-  "message.voicechat.ping_received_attempt": "%s번 시도하여 %sms 안에 응답을 수신함",
-  "message.voicechat.ping_retry": "응답이 없음. 다시 시도 중...",
-  "message.voicechat.ping_sent_waiting": "핑 전송함. 응답을 기다리는 중...",
-  "message.voicechat.ping_timed_out": "%s번 시도 이후 요청 시간 초과",
-  "message.voicechat.playback_unavailable": "재생할 수 없음",
-  "message.voicechat.player_no_voicechat": "%s은(는) %s을(를) 설치하지 않았습니다",
-  "message.voicechat.press_to_reassign_key": "키를 재할당하려면 누르세요",
-  "message.voicechat.processing_progress": "처리 중 %s%%",
-  "message.voicechat.processing_recording_session": "녹음 세션 처리 중...",
-  "message.voicechat.recording.disabled": "녹음을 시작하려면 누르세요",
-  "message.voicechat.recording.enabled": "녹음을 중단하려면 누르세요",
-  "message.voicechat.recording_disabled": "이 서버에선 녹음 기능을 사용할 수 없습니다",
-  "message.voicechat.recording_started": "녹음 시작",
-  "message.voicechat.recording_stopped": "녹음 중지",
-  "message.voicechat.save_session": "'%s'에 세션 저장 완료",
-  "message.voicechat.save_session_failed": "녹음 세션 저장 실패: %s",
-  "message.voicechat.saved_debug_report": "음성 채팅 디버그 기록이 %s에 저장되었습니다",
-  "message.voicechat.saved_debug_report_failed": "음성 채팅 디버그 기록 저장에 실패: %s",
-  "message.voicechat.search_empty": "해당 이름의 플레이어를 찾지 못했습니다",
-  "message.voicechat.search_hint": "검색...",
-  "message.voicechat.select": "선택",
-  "message.voicechat.select_microphone": "마이크 선택",
-  "message.voicechat.select_speaker": "스피커 선택",
-  "message.voicechat.sending_ping": "핑 전송 중...",
-  "message.voicechat.server_port": "%s 포트로 호스팅 된 음성 채팅 서버",
-  "message.voicechat.set_up": "설정하려면 %s을(를) 누르세요",
-  "message.voicechat.settings": "설정",
-  "message.voicechat.show_group_hud.disabled": "그룹 채팅 HUD를 표시하려면 누르세요",
-  "message.voicechat.show_group_hud.enabled": "그룹 채팅 HUD를 숨기려면 누르세요",
-  "message.voicechat.speaker_unavailable": "스피커를 사용할 수 없음",
-  "message.voicechat.storage_size": "예상 저장소 크기 %s",
-  "message.voicechat.voice_activation": "음성 입력 감도: %s dB",
-  "message.voicechat.voice_activation.disabled": "0 dB 마이크를 비활성화합니다!",
-  "message.voicechat.voice_chat_not_connected": "음성 채팅이 연결되지 않음",
-  "message.voicechat.voice_chat_unavailable": "음성 채팅을 할 수 없음",
-  "message.voicechat.voice_chat_volume": "음성 채팅 볼륨: %s",
-  "message.voicechat.volume_amplification": "증폭: %s",
-  "message.voicechat.wrong_password": "잘못된 암호",
-  "resourcepack.voicechat.black_icons": "검은색 아이콘",
-  "resourcepack.voicechat.black_icons.description": "검은색 음성 채팅 아이콘",
   "resourcepack.voicechat.classic_icons": "클래식 아이콘",
```

Comment:
Why are there so many changes?

Reply:
From the description of this pull request:
> 3. Translation file cleanup:
>    - This includes **rearranging all lines so that they are in the same order as in the source U.S. English translation file**, removing translation keys that are no longer in use, and unifying line endings to always use LF only.
>    - Most of these changes **were made automatically by a script I created**, so the chance of error is minimal.

It looks like the line order in the Korean translation was completely messed up and is now being restored to how it should have been from the beginning. The same goes for the Polish translation you commented on below, as well as all the others.

You can use the following command in Bash to verify that nothing has really changed, apart from what I've already written about in the quoted text above:
```bash
diff -u --strip-trailing-cr --color <(sort old.json) <(sort new.json)
```

Reply:
Oh nice, I'll trust you there.